electron/legendary/games.ts

```diff
@@ -310,11 +314,8 @@ Categories=Game;
    * @public
    */
   public async removeDesktopShortcut() {
-    if (process.platform !== 'linux'){
-      return
-    }
     const gameInfo = await this.getGameInfo()
-    const desktopFile = `${home}/Desktop/${gameInfo.title}.desktop`
+    const desktopFile = `${app.getPath('desktop')}/${gameInfo.title}.desktop`
```

Comment:
This is the desktop path, not the applications folder.
I think its possible to run `shell: {something}` to get the start menu folder.

Reply:
I know. app.getPath('desktop') points to the same directory, but it's more cross platform. It supports every language.

Reply:
No, i mean, this is not the start menu folder. This is still the desktop folder